.storybook/DocsDecorator/HealthFactors/Icons/index.styles.ts

```diff
@@ -0,0 +1,17 @@
+import { css, Theme } from '@emotion/react';
+import pixelsToRem from '../../../../src/app/utilities/pixelsToRem';
+
+const styles = {
+  icon: (theme: Theme) =>
+    css({
+      display: 'block',
+      fill: 'currentcolor',
+      [theme.mq.HIGH_CONTRAST]: {
+        svg: {
+          fill: 'VisitedText',
```

Comment:
Why 'visited'?

Reply:
I never used fill with attributes that are not colours, so I am unsure what is available to me. I will experiment with this today and find a working attribute.

Reply:
FY, I found this documentation about high contrast mode styling https://blogs.windows.com/msedgedev/2020/09/17/styling-for-windows-high-contrast-with-new-standards-for-forced-colors/
Might be good to know 🤷‍♂️